src/System/Led.cpp

```diff
@@ -5,57 +5,57 @@ Led::Led(uint16_t pin, uint8_t brightness){
     _brightness = brightness;
     // Enable output pin
     pinMode(pin, OUTPUT);
-    _state = OFF;
+    _state = LED_OFF;
     off();
 }
 
 void Led::handle(){
     uint16_t current_time_in_interval = (millis() - _interval_time_start) % _interval;
 
-    if(_state == FLASH_ON){ 
+    if(_state == LED_FLASH_ON){ 
         if(current_time_in_interval > _interval / 2 ){
-            _state = FLASH_OFF;
+            _state = LED_FLASH_OFF;
             analogWrite(_pin, 0);
         }
-    }else if(_state == FLASH_OFF){
+    }else if(_state == LED_FLASH_OFF){
         if(!_flash_count_left){
-            _state = OFF;
+            _state = LED_OFF;
         }else if(current_time_in_interval <= _interval / 2){
-            _state = FLASH_ON;
+            _state = LED_FLASH_ON;
             _flash_count_left--;
             analogWrite(_pin, _brightness);
         }
-    }else if(_state == FLASH_REPEAT){
+    }else if(_state == LED_FLASH_REPEAT){
         if(current_time_in_interval > _interval / 2 ){
             analogWrite(_pin, 0);
         }else{
             analogWrite(_pin, _brightness);
         }
     }
-    else if(_state == PULSE){
+    else if(_state == LED_PULSE){
         uint8_t pulse_brightness;
         if(current_time_in_interval < _interval / 2){
             pulse_brightness = (_brightness *current_time_in_interval * 2) / _interval;
         }else{
             pulse_brightness = (_brightness * (_interval - current_time_in_interval) * 2) / _interval;
         }
 
```

Comment:
This can be rewritten as a switch statement; would make the code a bit cleaner.

Reply:
Fixed!